### ISSUE TITLE
module 'audio_session' not found

### ISSUE THREAD

**Issue by ginfo2021** [ryanheise/audio_session#38]
We are getting this error once we install audio_session plugin -

Xcode's output:
↳
    /Users/birendrayadav/Desktop/wemeetnew/wemeet/ios/Runner/GeneratedPluginRegistrant.m:10:9:
    fatal error: module 'audio_session' not found
    @import audio_session;
     ~~~~~~~^~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete

Could not build the application for the simulator.
Error launching application on iphone.

**Comment by ryanheise** [comment]
Are you able to provide a minimal reproduction project?

**Comment by ginfo2021** [comment]
when we run this command "Run flutter create bug" and "cd bug" we are able to run the project but not from main project file.

"Flutter doctor -v"

[✓] Flutter (Channel stable, 2.2.3, on macOS 11.3.1 20E241 darwin-arm, locale en-IN)
    • Flutter version 2.2.3 at /Users/birendrayadav/Desktop/wemeetafrica/flutter
    • Framework revision f4abaa0735 (8 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.2

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] Connected device (1 available)
    • iPhone 8 (mobile) • 0557A4E6-B8FF-4304-B24A-DD7121D05A0A • ios • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)

! Doctor found issues in 3 categories.

**Comment by ryanheise** [comment]
Hopefully that may help you to fix it, since you can look at the difference between the working project and the broken project.

**Comment by ginfo2021** [comment]
Sorry I didn't get you, how we will able to fix this issue?

**Comment by ryanheise** [comment]
From what you told me, a newly created project will work fine, so you should just use that.

**Comment by ginfo2021** [comment]
new project without this plugin works fine but when include this plugin shows error, am working on macOS M1 chip is there any machine dependency for this plugin?

**Comment by ryanheise** [comment]
Does the example project in this repository also fail you?

**Comment by ryanheise** [comment]
Also, does this StackOverflow question help?

https://stackoverflow.com/questions/66691954/module-audio-session-not-found

**Comment by jtmuller5** [comment]
This is still not working. I tried with the newest version of just_audio and tried previous versions back until ^0.8.0. It was working a week or so ago so I'm not sure what changed.

**Comment by ryanheise** [comment]
Is that an answer to my previous question? i.e. Are you saying that the example project in the example subdirectory is not working?

**Comment by moneer-muntazah** [comment]
Having the same problem

**Comment by ryanheise** [comment]
Since nobody wants to provide a minimal reproduction project and is ignoring my requests, I see no other option but to live this issue. If anyone wants to see this issue actually fixed, then please open a new issue and provide all of the required information that was requested.
